pages/faq.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div id="main-content" class="text-wrapper">
+    <h1 class="text-center mb-6">FAQ with Professor Julian Goodare</h1>
+    <p class="mb-6 text-justify">
+      University of Edinburgh students, Ruby Imrie and Ellie Whitehead, sat down
```

Comment:
you can link to your section in the about section here.

Reply:
good idea, done

Reply:
The styling for the link appears to be missing. Is this expected? Thanks Ruby

Reply:
ahh I've just seen your comment. Will mark as resolved.